packages/mobile/src/components/now-playing-drawer/AudioControls.tsx

```diff
@@ -72,6 +73,12 @@ export const AudioControls = ({ onNext, onPrevious }: AudioControlsProps) => {
   const shuffleEnabled = useSelectorWeb(getShuffle)
   const repeatMode = useSelectorWeb(getRepeat)
 
+  const {
```

Comment:
is this for a different ticket?

Reply:
Yeah AUD-1767, I put it in the description. Could have made a separate PR but thought it was quick enough to knock out while in the now playing drawer

Reply:
ah i see! let me add a comment real quick then